go.mod

```diff
@@ -3,18 +3,15 @@ module knative.dev/net-contour
 go 1.16
 
 require (
-	github.com/go-openapi/jsonpointer v0.19.5 // indirect
-	github.com/go-openapi/jsonreference v0.19.5 // indirect
-	github.com/go-openapi/spec v0.19.6 // indirect
 	github.com/go-openapi/swag v0.19.15 // indirect
 	github.com/google/go-cmp v0.5.6
 	github.com/mailru/easyjson v0.7.7 // indirect
 	github.com/mikefarah/yq/v3 v3.0.0-20200601230220-721dd57ed41b
-	github.com/projectcontour/contour v1.18.2
+	github.com/projectcontour/contour v1.19.1
 	go.uber.org/zap v1.19.1
-	k8s.io/api v0.21.4
-	k8s.io/apimachinery v0.21.4
-	k8s.io/client-go v0.21.4
+	k8s.io/api v0.22.0
```

Comment:
Contour 1.19 requires k8s 1.22 deps but Knative still needs to use 1.21 deps until https://github.com/knative/pkg/pull/2327 is in. So we need to pin k8s libs in the `replace` field below like this:

```
replace (
        github.com/envoyproxy/go-control-plane => github.com/envoyproxy/go-control-plane v0.9.10-0.20210614203518-782de910ff04
        k8s.io/api => k8s.io/api v0.21.4
        k8s.io/apimachinery => k8s.io/apimachinery v0.21.4
        k8s.io/client-go => k8s.io/client-go v0.21.4
)
```

Reply:
Ah thanks for this! Will change it.